geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ClientHealthMonitor.java

```diff
@@ -59,18 +59,20 @@
 
   /**
    * The map of known clients
+   *
+   * Accesses must be locked by _clientHeartbeatsLock
    */
-  protected volatile Map _clientHeartbeats = Collections.EMPTY_MAP;
+  private Map<ClientProxyMembershipID, Long> _clientHeartbeats = Collections.emptyMap();
 
   /**
    * An object used to lock the map of known clients
    */
-  protected final Object _clientHeartbeatsLock = new Object();
+  private final Object _clientHeartbeatsLock = new Object();
```

Comment:
Perhaps remove the leading underscores on variables? Hungarian notation is the root of all evil.

Reply:
I could, but it's consistent with the other fields on this class, and I'm trying to keep this a quick, minimally invasive change.